packages/client/src/helpers/client/interceptors/authentication/AxiosAuthenticationTokenManager.ts

```diff
@@ -632,14 +676,14 @@ class AxiosAuthenticationTokenManager {
    *
    * @param expiredUserToken - The expired user token data.
    */
-  raiseOnUserSessionTerminatedEvent(expiredUserToken) {
+  raiseOnUserSessionTerminatedEvent(expiredUserToken: any) {
```

Comment:
Shouldn't this parameter be a `string`?

Reply:
Done. We might have missed it, it is now typed. Not a string, but an object :)